### ISSUE TITLE
Validation accuracy different during training and testing

### ISSUE THREAD

**Issue by VJatla** [open-mmlab/mmaction2#409]
Hello,

I am using mmaction2 to train on my own dataset which has two activity classes. I have seen that during training I am getting a validation accuracy (top_1) as 84% while after saving model and testing on save validation set I am getting accuracy of 75%. 

From the theory this should not happen (1-2%) is okay but 10% difference seems to be a lot. Could you please help me understand what might be happening here?

Regards
Vj

**Comment by innerlee** [comment]
You may check the sampling strategy, number of clips etc. You may copy these setting from val to test to debug

**Comment by VJatla** [comment]
Thank you. I will check and get back.

**Comment by dreamerlin** [comment]
Addition: The `top_k` value shown during training is for training dataset. Things may be different for validation and testing : )
